src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
 
 Moreover, it is also important to notice that the Scheduler and 3 microservices (*scheduling-api*, *job-planner*, and *proactive-cloud-watch*) have to point to the same scheduler database since they use that one.
 
+==== Example configuration with PostgreSQL
+It is recommanded to use a PostgreSQL database in a ProActive prod environment.
+In the following, we show an example of PostgreSQL installation and configuration to be used along with ProActive.
+
+===== PostgreSQL version
+PostgreSQL version 12.2 or 12.9 is required to be installed. PostgreSQL could be installed on the same machine as for ProActive or on a separate server (on-prem or on the cloud).
```

Comment:
PostgreSQL versions 12.0 to 12.11 are fully compatible with ProActive. More recent versions of PostgreSQL have not been fully tested yet for compatibility. 
PostgreSQL server can be installed on the same machine as the ProActive server or on a separate machine.